extension/persistence/eclipselink/src/main/java/io/polaris/extension/persistence/impl/eclipselink/PolarisEclipseLinkMetaStoreSessionImpl.java

```diff
@@ -146,16 +164,24 @@ private EntityManagerFactory createEntityManagerFactory(
       properties.put(ECLIPSELINK_PERSISTENCE_XML, confFile);
 
       return Persistence.createEntityManagerFactory(persistenceUnitName, properties);
-    } catch (Exception e) {
+    } catch (IOException e) {
```

Comment:
@collado-mike  Didn't get your original comment "catch RuntimeException separately and just rethrow".